composer.json

```diff
@@ -53,7 +53,7 @@
         "files": ["src/Magento/FunctionalTestingFramework/_bootstrap.php"],
         "psr-4": {
             "Magento\\FunctionalTestingFramework\\": "src/Magento/FunctionalTestingFramework",
-            "MFTF\\": "dev/tests/functional/MFTF"
+            "MFTF\\": "dev/tests/functional/tests/MFTF"
```

Comment:
In standalone setup, how to make it work when helper is used in app/code/Magento modules? I got a generation error.

Reply:
Verified it worked